corehq/apps/es/index/analysis.py

```diff
@@ -0,0 +1,37 @@
+
+DEFAULT_ANALYZER = {
```

Comment:
What do you think of renaming this to `LOWERCASE_WHITESPACE_ANALYZER`?

Maybe in a separate PR if it seems out of scope for this one?

Reply:
See 491c3f066a8